src/MyParcelComApi.php

```diff
@@ -151,6 +151,7 @@ public function getPickUpDropOffLocations(
         ?string $streetNumber = null,
         CarrierInterface $specificCarrier = null,
         bool $onlyActiveContracts = true,
+        ?array $filters = null,
         int $ttl = self::TTL_10MIN,
```

Comment:
To avoid possible breaking changes, we should add new parameters at the end of the signature.